src/services/housing.js

```diff
@@ -208,7 +250,11 @@ const changeApartmentAppEditor = async (applicationID, newEditorUsername) => {
  */
 const getApartmentApplication = async (applicationID) => {
   try {
-    return await http.get(`housing/apartment/applications/${applicationID}/`);
+    let applicationResult = await http.get(`housing/apartment/applications/${applicationID}/`);
+    if (applicationResult) {
+      await setApplicationDetails(applicationResult);
```

Comment:
For some reason, the Profile objects from this endpoint are lacking the `PersonType` prop, but all the other expected props appear to be there...

This breaks several parts of the page

Reply:
Ah yes, apparently that property is added by the `ProfilesController.Get()` method and isn't part of the actual view models. I'm not sure what the best solution for this is, outside of getting profiles from the ProfilesController instead of creating them in the backend.

Reply:
If we have the HousingController.GetApartmentApplication send only the usernames, and then have this file do a user.getProfileInfo for each, but this would mean a whole bunch of extra back-and-forth between the UI and API just to load a single application, and the API code needs to get the student profiles during the loading process regardless, so it would seem extremely inefficient. 
This sounds like something to be best addressed in the API side, I will make a note